Jenkinsfile

```diff
@@ -40,7 +40,7 @@ pipeline {
                         echo "Bootstrapping commit ${SG_COMMIT}"
                         sh 'cp .scm-checkout/bootstrap.sh .'
                         sh 'chmod +x bootstrap.sh'
-                        sh "./bootstrap.sh -p sg-accel -c ${SG_COMMIT}"
+                        sh "./bootstrap.sh -p sg -c ${SG_COMMIT}"
```

Comment:
Will need some way of passing down the fact we want to bootstrap the EE version here.

Previously we used to rely on the `-p sg-accel` flag for this, which would set the `-g all` flag on repo tool, which pulls in the "cb_sg_enterprise" group dependencies for the private `go-fleecedelta`.

Reply:
Good catch - switched bootstrap.sh to take an -e flag for ce vs. ee and updated Dockerfile/Jenkinsfile